src/test/java/seedu/linkedout/model/applicant/JobTest.java

```diff
@@ -30,7 +92,27 @@ public void isValidJob() {
 
         // valid jobs
         assertTrue(Job.isValidJob("Software Engineer"));
-        assertTrue(Job.isValidJob("-")); // one character
+        assertTrue(Job.isValidJob("J")); // one character
         assertTrue(Job.isValidJob("Full Stack Software Engineer")); // long job
     }
+
+    @Test
+    public void equals_thisJob_true() {
+        Job job = new Job("job");
+        assertTrue(job.equals(job));
+    }
+
+    @Test
+    public void equals_otherType_false() {
+        Job job = new Job("job");
+        String string = "job";
+        assertFalse(job.equals(string));
+    }
+
+    @Test
+    public void equals_otherJobSameValue_true() {
```

Comment:
is this typo, otherJob Same Type? and false?
```suggestion
    public void equals_otherJobSameType_False() {
```

Reply:
Good catch, will make the change accordingly.